<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fractal Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>fractal.ico</normaloff>fractal.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(153, 153, 153);</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0">
    <item>
     <widget class="View" name="view">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(102, 102, 102);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="panel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(102, 102, 102);
font: 12pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,3,2,3,1,2,3,3" rowminimumheight="0,0,0,0,0,0,0,0">
       <item row="4" column="3">
        <widget class="QDoubleSpinBox" name="s">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="use_s">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="h">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cx</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="use_h">
         <property name="text">
          <string/>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QPushButton" name="run">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="cy">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="value">
          <double>0.160000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Saturation</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="cx">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="value">
          <double>-0.800000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Hue</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cy</string>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QPushButton" name="save">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Center X</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QCheckBox" name="use_v">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="x">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="minimum">
          <double>-1.500000000000000</double>
         </property>
         <property name="maximum">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Center Y</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="y">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="minimum">
          <double>-1.500000000000000</double>
         </property>
         <property name="maximum">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QDoubleSpinBox" name="v">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>10</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QSpinBox" name="loop">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Loop Limit</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QSpinBox" name="size">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>800</number>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="processing_time">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Processing Time</string>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <widget class="QDoubleSpinBox" name="scale">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(51, 51, 51);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000000000000007517448691651820862747142906435240821348290910235776592524241520466454110109775803542826595503885252632667750400.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
